### ISSUE TITLE
no transfer of contact photos

### ISSUE THREAD

**Issue by Stefan110755** [zanonmark/Google-4-TbSync#59]
all data was transferred from my Google account to TB, only the contact photos were not. Is this standard or am I doing something wrong?

**Comment by zanonmark** [comment]
Hi Stefan,

it's perfectly fine: that feature is still missing, it's in my TODO-list, but only after a full-WebExtension version will be released.

Closing because it's a duplicate of #46.

Thanks,
MZ
